### PR TITLE
497 - improve fees dataset

### LINKED ISSUE #497: Improve fees dataset
Store level and cycle in the `fees` table.

### PR DESCRIPTION
**Pull request by piotrkosecki**
resolves #497 

### Database changes:
In table `fees` two new columns were added:
```
    cycle integer,
    level integer
```

### CODE REVIEW

File: src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala [hunk 1]
```diff
@@ -39,13 +39,7 @@ object DatabaseConversions {
...
   implicit val averageFeesToFeeRow = new Conversion[Id, AverageFees, Tables.FeesRow] {
     override def convert(from: AverageFees) =
-      Tables.FeesRow(
-        low = from.low,
-        medium = from.medium,
-        high = from.high,
-        timestamp = from.timestamp,
-        kind = from.kind
-      )
+      from.into[Tables.FeesRow].transform
```
Comment (ivanopagano) [inline src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala]: ❤️

File: src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala [hunk 2]
```diff
@@ -953,7 +953,9 @@ class TezosDatabaseOperationsTest
           medium = mu,
           high = mu + sigma,
           timestamp = latest,
-          kind = ops.head.kind
+          kind = ops.head.kind,
+          cycle = None,
+          level = Some(0)
```
Comment (ivanopagano) [inline src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala]: Are we testing here that actual latest value is the one used in the average result?

File: src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala (hunk 1 above)
Comment (dorians) [reply src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala]: good one!

File: src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala (hunk 2 above)
Comment (piotrkosecki) [reply src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTest.scala]: no, we're not, those values were generated, I'm not why they are not `Nones`, as they are optional in the table

**Comment by ivanopagano** [comment]
Now master has the sql schema synced with the slick models, you can merge it to get a consistent story of the db

**Pull request merged**